src/RecyclableMemoryStreamManager.cs

```diff
@@ -319,83 +153,215 @@ public long LargeBuffersFree
         }
 
         /// <summary>
-        /// How many bytes of small free blocks to allow before we start dropping
-        /// those returned to us.
+        /// Parameters for customizing the behavior of <see cref="RecyclableMemoryStreamManager"/>
+        /// </summary>
+        public struct Options
```

Comment:
We might want to make `Options` a class so it will play nice with `IOptions` which is defined as `public interface IOptions<out TOptions> where TOptions : class`